enclave-runtime/src/stf_task_thread.rs

```diff
@@ -117,7 +117,7 @@ fn run_extrinsic_request_daemon_internal() -> Result<()> {
 	let stf_enclave_signer =
 		Arc::new(EnclaveStfEnclaveSigner::new(state_observer, ocall_api, shielding_key_repository));
 
-	let request_context = RequestContext::new(
+	let request_context = VerifyContext::new(
```

Comment:
Can we have a `stf_task_receiver.rs` to handle the stf-tasks?
Then inside the thread it should as simple as launching the receiver

Reply:
I have thought about creating a `stf_task_receiver.rs` and move it to `litentry/core/stf_task_handler/`. But it came to the same problem as above of recursive include. 
Or do you mean simply split into two files, but still sit inside `enclave-runtime` ?